operator/pkg/karmadaresource/webhookconfiguration/mainfests.go

```diff
@@ -0,0 +1,165 @@
+package webhookconfiguration
+
+const (
+	// KarmadaWebhookMutatingWebhookConfiguration is karmada webhook mutatingWebhookConfiguration manifest
+	KarmadaWebhookMutatingWebhookConfiguration = `
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-config
+  labels:
+    app: mutating-config
+webhooks:
+  - name: propagationpolicy.karmada.io
+    rules:
+      - operations: ["CREATE", "UPDATE"]
+        apiGroups: ["policy.karmada.io"]
+        apiVersions: ["*"]
+        resources: ["propagationpolicies"]
+        scope: "Namespaced"
+    clientConfig:
+      url: https://karmada-webhook.{{ .Namespace }}.svc:443/mutate-propagationpolicy
```

Comment:
```
E0228 09:20:16.187743       1 work.go:64] Failed to create/update work karmada-es-member68/karmada-impersonator-7cbb6bd5c9. Error: Internal error occurred: failed calling webhook "work.karmada.io": failed to call webhook: Post "https://karmada-webhook.karmada-system.svc:443/mutate-work?timeout=3s": dial tcp: lookup karmada-webhook.karmada-system.svc on 10.96.0.10:53: no such host
E0228 09:20:16.187784       1 unified_auth_controller.go:126] failed to sync impersonate clusterrole to cluster(member68): Internal error occurred: failed calling webhook "work.karmada.io": failed to call webhook: Post "https://karmada-webhook.karmada-system.svc:443/mutate-work?timeout=3s": dial tcp: lookup karmada-webhook.karmada-system.svc on 10.96.0.10:53: no such host
E0228 09:20:16.187804       1 unified_auth_controller.go:75] Failed to sync impersonation config for cluster member68. Error: Internal error occurred: failed calling webhook "work.karmada.io": failed to call webhook: Post "https://karmada-webhook.karmada-system.svc:443/mutate-work?timeout=3s": dial tcp: lookup karmada-webhook.karmada-system.svc on 10.96.0.10:53: no such host.
E0228 09:20:16.187936       1 controller.go:326] "Reconciler error" err="Internal error occurred: failed calling webhook \"work.karmada.io\": failed to call webhook: Post \"https://karmada-webhook.karmada-system.svc:443/mutate-work?timeout=3s\": dial tcp: lookup karmada-webhook.karmada-system.svc on 10.96.0.10:53: no such host" controller="cluster" controllerGroup="cluster.karmada.io" controllerKind="Cluster" Cluster="member68" namespace="" name="member68" reconcileID=c701ed51-2714-4cdf-87ab-f5f131532873
I0228 09:20:16.188042       1 recorder.go:103] "events: Internal error occurred: failed calling webhook \"work.karmada.io\": failed to call webhook: Post \"https://karmada-webhook.karmada-system.svc:443/mutate-work?timeout=3s\": dial tcp: lookup karmada-webhook.karmada-system.svc on 10.96.0.10:53: no such host" type="Warning" object={Kind:Cluster Namespace: Name:member68 UID:f19f6117-914f-44fb-9579-5a86a19caa84 APIVersion:cluster.karmada.io/v1alpha1 ResourceVersion:643682 FieldPath:} reason="SyncImpersonationConfigFailed"
```

The reason seems that it fix the svc is `karmada-webhook`, but the generated webhook is `karmada-demo-webhook`

```
[root@master67 config]# kubectl get svc -n karmada-system
NAME                                TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
karmada-demo-aggregated-apiserver   ClusterIP   10.101.141.83   <none>        443/TCP             4d19h
karmada-demo-apiserver              NodePort    10.107.224.49   <none>        5443:32443/TCP      4d19h
karmada-demo-etcd                   ClusterIP   None            <none>        2379/TCP,2380/TCP   4d19h
karmada-demo-etcd-client            ClusterIP   10.100.147.41   <none>        2379/TCP            4d19h
karmada-demo-webhook                ClusterIP   10.98.217.253   <none>        443/TCP             4d19h
```

Reply:
Thanks for pointing out the error.

Reply:
@lonelyCZ Amended.